test/e2e/specs/test.spec.js

```diff
@@ -188,18 +222,24 @@ describe('Wallet App Test Cases', { execTimeout: DEFAULT_EXEC_TIMEOUT }, () => {
       cy.contains('Arguments').should('be.visible');
     });
 
-    it('should cancel the bid by price and verify IST balance', () => {
-      cy.getTokenAmount('IST').then((initialTokenValue) => {
-        cy.visit('/wallet/');
-        cy.contains('Exit').click();
-        cy.acceptAccess().then((taskCompleted) => {
-          expect(taskCompleted).to.be.true;
-        });
-        cy.contains('Accepted', { timeout: DEFAULT_TIMEOUT }).should('exist');
-        cy.wait(5000);
-        cy.getTokenAmount('IST').then((tokenValue) => {
-          expect(tokenValue).to.greaterThan(initialTokenValue);
-        });
+    it('should cancel the bid by price', () => {
+      cy.visit('/wallet/');
+      cy.contains('Exit').click();
+      cy.acceptAccess().then((taskCompleted) => {
+        expect(taskCompleted).to.be.true;
+      });
+      cy.contains('Accepted', { timeout: DEFAULT_TIMEOUT }).should('exist');
+    });
+
+    it("should see increase in user1's IST balance after canceling bid", () => {
+      cy.wait(QUICK_WAIT);
```

Comment:
Usually a hardcoded wait cause flakiness. Have we tweaked this number?
Btw can we think of polling on the bid tx? Can await on a promise which will be resolved once we get success of tx.

Reply:
Initially, the wait was 5 seconds. I increased to 10 seconds and things have been smooth so far in my testing. 

> Btw can we think of polling on the bid tx?

I am not aware of how to do that. But a custom command that does that can be a valuable addition to our e2e tests.
We can discuss the mechanism of doing that and I can add that in a follow-up PR?

Reply:
Surely yes 👍